packages/venia-concept/src/queries/getCategoryList.graphql

```diff
@@ -1,15 +1,24 @@
 # TODO: get only active categories from graphql when it is ready
-query categoryList($id: Int!) {
+
+query categoryList($id: Int!, $forNavigation: Boolean!) {
     category(id: $id) {
         id
+        name @include(if: $forNavigation)
```

Comment:
Would have loved to use named fragments here, but Magento doesn't currently support them (magento/graphql-ce#147). Sorry for the verbose directives :confused:

Reply:
Agreed. We really need fragments.